graphene_django/types.py

```diff
@@ -191,9 +191,5 @@ class ErrorType(ObjectType):
 
     @classmethod
     def from_errors(cls, errors):
-        data = (
-            camelize(errors)
-            if graphene_settings.DJANGO_GRAPHENE_CAMELCASE_ERRORS
-            else errors
-        )
+        data = camelize(errors) if graphene_settings.CAMELCASE_ERRORS else errors
         return [ErrorType(field=key, messages=value) for key, value in data.items()]
```

Comment:
BTW I forgot to change `ErrorType` to `cls` here.

Reply:
Thanks @kalekseev . Updated.